README.md

```diff
@@ -10,7 +10,7 @@ It checks for pull requests dependencies, both same-repo and external, which sho
 
 Pierre will perform checks only upon PR creation and Comment Activity (added/removed). In every case it will fetch all the PR's bodies (the PR body itself and from all its comments), extract the dependencies and perform the checks. Thus, it **does not** observe the dependencies themselves and re-run the checks if their status change.
 
-For now, the best way of re-checkind the dependencies statuses is to add a new comment. I suggest `pierre re-check`. :smiley:
+For now, the best way of re-checking the dependencies statuses is to add a new comment. I suggest `pierre re-check`. :smiley:
```

Comment:
in the future please split different unconnected changes into separate PRs

Reply:
Sorry, @mosheeshel . I though that the earlier request has already been merged.

Reply:
It was not merged, you simply closed the PR. I'm currently the only one that can merge PRs, so a contributor should only check for any outstanding comments and wait for approval. Once it's approved I can merge it.